invisible_cities/cities/beersheba.py

```diff
@@ -64,6 +63,31 @@
 from .. core                   import system_of_units as units
 
 
+# Temporary. The removal of the event model will fix this.
+from collections import defaultdict
+def hitc_to_df_(hitc):
+    columns = defaultdict(list)
+    for hit in hitc.hits:
+        columns["event"   ].append(hitc.event)
+        columns["time"    ].append(hitc.time)
+        columns["npeak"   ].append(hit .npeak)
+        columns["Xpeak"   ].append(hit .Xpeak)
+        columns["Ypeak"   ].append(hit .Ypeak)
+        columns["nsipm"   ].append(hit .nsipm)
+        columns["X"       ].append(hit .X)
+        columns["Y"       ].append(hit .Y)
+        columns["Xrms"    ].append(hit .Xrms)
+        columns["Yrms"    ].append(hit .Yrms)
+        columns["Z"       ].append(hit .Z)
+        columns["Q"       ].append(hit .Q)
+        columns["E"       ].append(hit .E)
+        columns["Qc"      ].append(hit .Qc)
+        columns["Ec"      ].append(hit .Ec)
+        columns["track_id"].append(hit .track_id)
+        columns["Ep"      ].append(hit .Ep)
+    return pd.DataFrame(columns)
```

Comment:
The previous Beersheba version worked directly on dataframes, not on HitCollections. However, the thresholding step introduced in the new version works with the latter object.
Because the thresholding is also used in Esmeralda (where the use of HitCollection is too extended to be removed) and the plan is to remove the HitCollection object altogether, I decided to do this transformation to be able to use the existing thresholding implementation in Beersheba.